tests/test_templater.py

```diff
@@ -291,3 +292,26 @@ def test_set_template_verbosity():
     outcome = ''
     # Check that only the correct messages were logged
     assert result == outcome
+
+def test_log_passing():
+    """Unit test for checking application of the logger object passing"""
+
+    input_file = os.path.join(uwtools_file_base, "fixtures/nml.IN")
+    logfile = os.path.join(os.path.dirname(templater.__file__), "templater.log")
+    ref = "Only the templater logger would see this"
```

Comment:
What happens here if we want to run two templater tests simultaneously?

Reply:
Good point, I pulled that from my verbosity test, which should also be changed to use `tmp_path`. As-is, if two tests run simultaneously, they'll write to the same log file, which shouldn't cause an error despite it, but is bad manners at least. However, looking back through, templater and set_config both also hard code a fixed file, which will lead to the same problem if two users run from the same install or, more likely, the stack. I'll add an issue to address that

Reply:
Removed the logfile logic and replaced with stdout